responders/src/API/EMBC.Responders.API/Controllers/RegistrationsController.Search.cs

```diff
@@ -104,8 +104,8 @@ public EvacuationSearchMapping()
                 .ForMember(d => d.Status, opts => opts.MapFrom(s => s.Status))
                 .ForMember(d => d.HouseholdMembers, opts => opts.MapFrom(s => s.NeedsAssessment.HouseholdMembers))
                 .ForMember(d => d.EvacuatedFrom, opts => opts.MapFrom(s => s.EvacuatedFromAddress))
-                .ForMember(d => d.CreatedOn, opts => opts.Ignore())
-                .ForMember(d => d.ModifiedOn, opts => opts.Ignore())
+                .ForMember(d => d.CreatedOn, opts => opts.MapFrom(s => s.EvacuationDate))
```

Comment:
There should be specific fields for create and modify dates, I think @josephweinkamgov is working on adding them

Reply:
Sorry, Monday mind reset - the fields exist on NeedsAssessment, the mapping should use these fields to populate the created and modified timestamps